README.md

```diff
@@ -98,6 +98,10 @@ require'nvim-treesitter.configs'.setup {
     highlight = {
         enable = true,                    -- false will disable the whole extension
         disable = { 'c', 'rust' },        -- list of language that will be disabled
+        custom_captures = {              -- mapping of user defined captures to highlight groups
+          ----- highlight own capture @foo.bar with highlight group "Identifier", see :h nvim-treesitter-query-extensions
+          -- ["foo.bar"] = "Identifier"
```

Comment:
I am a bit touchy here, but can we make that the same as above ?

Reply:
Are you talking about the formatting?

Reply:
No just uncommenting this, in the end, this is just an example config.